### PR TITLE
Visitor can adjust cart quantity; closes #13

### LINKED ISSUE #13: Adjusting the quantity of an item in the cart
Background: My cart has an item in it
      As a visitor
      When I visit "/cart"
      Then I should see my item with a quantity of 1
      And when I increase the quantity
      Then my current page should be '/cart'
      And that item's quantity should reflect the increase
      And the subtotal for that item should increase
      And the total for the cart should match that increase
      And when I decrease the quantity
      Then my current page should be '/cart'
      And that item's quantity should reflect the decrease
      And the subtotal for that item should decrease
      And the total for the cart should match that decrease

### PR DESCRIPTION
**Pull request by ksk5280**
User Story
Background: My cart has an item in it
      As a visitor
      When I visit "/cart"
      Then I should see my item with a quantity of 1
      And when I increase the quantity
      Then my current page should be '/cart'
      And that item's quantity should reflect the increase
      And the subtotal for that item should increase
      And the total for the cart should match that increase
      And when I decrease the quantity
      Then my current page should be '/cart'
      And that item's quantity should reflect the decrease
      And the subtotal for that item should decrease
      And the total for the cart should match that decrease

### CODE REVIEW

File: app/controllers/cart_items_controller.rb [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
   def create
     @item = Item.find(params[:item_id])
-    @cart.add_item(@item.id)
+    if params[:subtract]
+      @cart.remove_item(@item.id)
+      flash[:danger] = "#{@item.title} removed from cart!"
+    else
+      @cart.add_item(@item.id)
+      flash[:success] = "#{@item.title} added to cart!"
+    end
```
Comment (ksk5280) [inline app/controllers/cart_items_controller.rb]: @rwarbelow @Carmer @erinnachen @brennanholtzclaw 
What is the best way to move this logic out of the controller?

Comment (Carmer) [inline app/controllers/cart_items_controller.rb]: Think about a Cart method that can update and will take in the Params and the @item

**Comment by ksk5280** [comment]
Change cart_items path to cart

**Comment by scottfirestone** [comment]
Looks good! Merging...

**Pull request merged**